benchmarking/README.md

```diff
@@ -36,3 +36,78 @@ python examples/offline_inference_tt.py --measure_perf --max_seqs_in_batch 32 --
 - `--max_seqs_in_batch` (default: `32`):
   - **Maximum batch size** for inference, determining the number of prompts processed in parallel.
 
+### Online Benchmarking
+
+#### single user
+
+```bash
+python utils/prompt_client_cli.py \
```

Comment:
Does this give any performance metrics back for the single user?

Reply:
It prints them to stdout, I think this will be simpler to add to the `run_vllm_api_server.py` script so it covers that case as well. This could be done by setting the `--request-rate` for that benchmark to a number higher than e2e request latency.